src/python/scripts/dx-docker

```diff
@@ -210,9 +210,11 @@ def run(args):
 
     container_cmd = entrypoint + cmd
 
-    workdir = args.workdir
+    workdir = "/"
     if 'workingDirectory' in imagemeta['app']:
        workdir = imagemeta['app']['workingDirectory']
+    if args.workdir:
```

Comment:
Hi @geetduggal there's a new problem which is that `args.workdir` has a default setting of `/` so this condition is always true. Probably there should be no default value of `args.workdir`.

This broke viral-ngs for @dpark @tomkinsc I'm afraid...eg

```
$ dx-docker run -v /home/dnanexus:/user-data broadinstitute/viral-ngs:1.13.3 metagenomics.py
Image Repo      broadinstitute/viral-ngs
Image Tag       1.13.3
Image Size      3.6G
Image ID        sha256:1466e5a40aa0bd6b68b2285954bcd33002d19bb9e1ba09c3807a93fa3337bcd6
Last Updated    2016-12-13T16:21:10.093282599Z (33d 8h ago)
Registry        registry-1.docker.io
ENTRYPOINT      ["./env_wrapper.sh"]


/bin/sh: 1: ./env_wrapper.sh: not found
```

Can this be hotfixed?

Side comment, perhaps WORKDIR is worth showing in the log message too.